dotCMS/src/integration-test/java/com/dotcms/datagen/MultiTreeDataGen.java

```diff
@@ -0,0 +1,71 @@
+package com.dotcms.datagen;
+
+import com.dotmarketing.beans.MultiTree;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.htmlpageasset.model.HTMLPageAsset;
+
+public class MultiTreeDataGen extends AbstractDataGen<MultiTree> {
+    private Container container;
+    private Contentlet contentlet;
+    private String instanceID;
+    private String personalization;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'personalization' could be replaced by a local variable.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5411384)